hackathon_site/dashboard/frontend/src/components/dashboard/TeamCardAdmin/TeamCardAdmin.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import styles from "./TeamCardAdmin.module.scss";
+import Container from "@material-ui/core/Container";
+import Typography from "@material-ui/core/Typography";
+import Card from "@material-ui/core/Card";
+
+import TitledPaper from "components/general/TitledPaper/TitledPaper";
+
+interface CardProps {
+    team_name?: string;
+    members?: string[];
+}
+
+export const TeamCardAdmin = ({ team_name, members }: CardProps) => {
+    const title = team_name === null ? "No Team" : "Team " + team_name;
```

Comment:
since teamCode is required, you can remove this line

Reply:
removed